hw2/03-count.html

```diff
@@ -6,19 +6,20 @@
     <title>03 - Count</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="03-count.js"></script>
   </head>
-  <body>
-    <main>
+  <body class="row bg-dark justify-content-center">
+    <main class="bg-light w-75 mt-5">
       <h1>Exercise 03 - Count</h1>
```

Comment:
The header needs to be centered in order to match the reference screenshot.
It could also use a margin top.